### DISCUSSION TITLE
Using Github codespaces for a large tutorial?

### DISCUSSION THREAD

**Discussion by fpvandoorn** [community/community#57176] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I'm planning to use Github codespaces for a pretty large tutorial (~200 participants) next week, and most of the participants will have to create a new Github account to use codespaces. Does Github take any action against many (new) users that suddenly start using Github codespaces from the same location? 
A bit of context: it is at an academic conference for an open-source project, and the tutorial is only 3 hours long, so as long as we can use the standard time in a free account, we should be good.

**Comment by thomas-sickert** [comment, marked as answer]
Hey @fpvandoorn, excited to hear that you'll be using Codespaces for the tutorial!

GitHub uses a number of systems both automated and manual to detect abuse on the platform. While we can’t promise nobody will run into one or more of these systems when creating an account during the event it’s unlikely that genuine people adhering to our Acceptable Use Policies and ToS will run into trouble here.
